raglweb/__init__.py

```diff
@@ -284,6 +284,18 @@ def player(profile_id):
 def info():
     return render_template('info.html', start_time=_cfg['start_time'])
 
+def _make_division_prefix(division_title):
+    """Convert the human readable division title into a prefix for use in the filename."""
+    words = division_title.upper().split(' ')
+    prefix = 'RAGL-S10-' + words[0]
+    if prefix.endswith('S'):
+        prefix = prefix[:-1]
+    for word in words[1:]:
+        if len(word) > 0:
+            prefix += word[0]
+    if len(prefix) > 0:
+        prefix += '-'
```

Comment:
I don't think the condition is necessary

Reply:
Ah - true. I was trying to cover the case where there was no division given, but it's not really necessary.